OpenDF-Describe-Image/src/Server/server.py

```diff
@@ -62,36 +64,14 @@ def allowed_file(filename):
 
 
 @app.route('/api/analyze/<path:path>', methods=['GET'])
-def uploadImage(path):
-	IPath = "/"+path
-	#print(IPath)
+def upload_image(path):
+	IPath = get_actual_path(path)
+	print (IPath)
 	if(os.path.isfile(IPath)):
-		#return jsonify({"predicted" : IPath})
-		#process = subprocess.call("python ../main/main.py --model VGG16 --image " + IPath, shell=True)
-		value = predictImage("VGG16",IPath)
-		#print(process)
+		value = predict_image("VGG16",IPath)
+	else:
+		value = "File not found"
```

Comment:
Can we send a HTTP Error response here? 
You can do something like this  for now,
```
response = jsonify({'message': 'File not found'})
response.status_code = 400
return response
```

Later, we will add proper Exception handlers with `@app.errorhandler` annotations.